### ISSUE TITLE
[BUG] error: "VpcPeeringConnection" has no attribute "requester_vpc"; maybe "requester_vpc_info"?

### ISSUE THREAD

**Issue by lithammer** [vemel/mypy_boto3_builder#22]
**Describe the bug**
According the boto3 documentation, a [`VpcPeeringConnection`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#vpcpeeringconnection) has 'references' to [`accepter_vpc`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.VpcPeeringConnection.accepter_vpc) and [`requester_vpc`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.VpcPeeringConnection.requester_vpc). But these references aren't being generated so MyPy complains.

**To Reproduce**
1. Install 'boto3-stubs[ec2]'
2.  Run mypy on the following code sample

```python
import boto3

ec2 = boto3.resource("ec2", region_name="eu-north-1")
vpc = ec2.Vpc("foo")
vpc_peer = vpc.request_vpc_peering_connection(PeerVpcId="bar")
vpc_peer.accepter_vpc  # mypy error
vpc_peer.requester_vpc  # mypy error
```
3. Actual mypy output

```
bug.py:6: error: "VpcPeeringConnection" has no attribute "accepter_vpc"; maybe "accepter_vpc_info"?
    vpc_peer.accepter_vpc
    ^
bug.py:7: error: "VpcPeeringConnection" has no attribute "requester_vpc"; maybe "requester_vpc_info"?
    vpc_peer.requester_vpc
    ^
Found 2 errors in 1 file (checked 1 source file)
```

**Expected mypy-output**

```
Success: no issues found in 1 source file
```

**Additional context**
Your OS, `boto3-stubs` installation method, `boto3` version, etc.

OS: macOS and Linux (Alpine)
Installation method: `pip install boto3-stubs[ec2]`

```
boto3                               1.13.6
boto3-stubs                         1.13.6.0
mypy                                0.770
mypy-boto3                          1.13.6.0
mypy-boto3-ec2                      1.13.6.0
```

**Comment by vemel** [comment]
Thank you for the report, will take a look today.

**Comment by vemel** [comment]
Okay, looks like I do not parse resource's available references. This will take some time as I have to take a look and `botocore` shapes. Will keep you updated.

**Comment by lithammer** [comment]
No worries. Take your time 🙂

**Comment by vemel** [comment]
Already in master. Actually, this affects quite a lot of services, so good catch! I modified my tests a bit to check that references are working as they should. I hope it will be released today or tomorrow before new boto3 version release.

**Comment by vemel** [comment]
Released, included to `boto3_stubs>=1.13.25.0`. Thank you for the report.
